### ISSUE TITLE
Package dependency 'echarts4r'

### ISSUE THREAD

**Issue by Becker-Asano** [JohnCoene/coronavirus#15]
Hi,
after finally getting everything up and running locally with "RStudio" under Win10, the Shiny-Window opens and I read the following error message: 
> ERROR: es gibt kein Paket namens ‘echarts4r’.

Thus, I ran `install.packages("echarts4r")` and then rerun the following script, using 'Ctrl-Shift-Enter':

> library(coronavirus)
> virus <- crawl_coronavirus()
> run_app(virus)
> run_api()

(At least in my RStudio-Installation under Win10 I have to use "Source with Echo" to get this going!)

![Shiny_error](https://user-images.githubusercontent.com/1073833/76680590-0fcbf980-65ea-11ea-8c82-38c2ae1757ea.JPG)
The result shows some more errors with this output in the console:
`
Warnung: Error in : 'e_hide_grid_lines' ist kein von 'namespace:echarts4r' exportiertes Objekt
  110: stop
  109: getExportedValue
  108: ::
  107: function_list[[i]]
  106: freduce
  105: _fseq
  104: eval
  103: eval
  101: %>%
  100: mode_new_cases_echarts
   99: func
   86: origRenderFunc
   85: output$new_cases-plot
    5: runApp
    3: print.shiny.appobj
    1: source
261195358357 :  2
77144044928 :  2.910765
67902959968 :  2.933392
[snip]
`
In summary, it seems difficult to get this running under Win10 with RStudio, but I appreciate the scripting very much!

**Comment by JohnCoene** [comment]
Apologies, this is an issue with versioning on my end, will try to fix soon.

Reinstall echarst4r from Github and it should work.

```r
# install.packages("remotes")
remotes::install_github("JohnCoene/echarts4r")
````
Let me know, apologies.

**Comment by Becker-Asano** [comment]
Hi, thanks for the fast response. Did the following:

> .libPaths()
> [1] "C:/Users/Christian/Documents/R/win-library/3.6" "C:/Program Files/R/R-3.6.2/library"

Removed the echarts4r from folder C:/Program Files/R/R-3.6.2/library by calling `remove.packages("echarts4r")`.

> remotes::install_github("JohnCoene/echarts4r")
Downloading GitHub repo JohnCoene/echarts4r@master
√  checking for file 'C:\Users\Christian\AppData\Local\Temp\RtmpCmnhSD\remotes340e033341623\JohnCoene-echarts4r-c95466a/DESCRIPTION' ...
> -  preparing 'echarts4r': (38.7s)
> √  checking DESCRIPTION meta-information ...
> -  checking for LF line-endings in source and make files and shell scripts (1.9s)
> -  checking for empty or unneeded directories
> -  building 'echarts4r_0.3.0.9000.tar.gz'
> [snip]
   
Installing package into ‘C:/Users/Christian/Documents/R/win-library/3.6’, see above. Starting script as described in first comment.

Only one warning 

> Warnung: Error in loadNamespace: es gibt kein Paket namens ‘echarts4r.maps’

but Shiny looks fine:

![Shiny_resolved](https://user-images.githubusercontent.com/1073833/76681669-bc5fa880-65f5-11ea-8914-55383bb3e992.JPG)

Thank you!

**Comment by JohnCoene** [comment]
Thank you, strange, if you get issues with maps install it from Github. Sorry again, will try to fix all of this soon.

```r
remotes::install_github('JohnCoene/echarts4r.maps')
```

**Comment by Becker-Asano** [comment]
Thanks again. Now the map shows as well:

![Shiny_maps_resolved](https://user-images.githubusercontent.com/1073833/76681756-b9b18300-65f6-11ea-8107-8f5a8998078a.JPG)

**Comment by Becker-Asano** [comment]
BTW: The `create_config()` function creates the `_coronavirus.yml` file under Win10 in "%HOMEPATH%/Documents" (see `getwd()` output) not in any of the R-related folders. Took me some time.

**Comment by JohnCoene** [comment]
Ah, so it does not create it in the directory wherefrom you ran it?

**Comment by Becker-Asano** [comment]
No. Unfortunately, not. It is also searched for and opened from the Documents folder.

**Comment by JohnCoene** [comment]
Thank you, will look into this.
